IaC/outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "DBHost" {
+  value = aws_db_instance.MTBZoneDB.address
+}
+
+output "CatalogAPIUrl" {
```

Comment:
Two things :
* All `output`s coming from modules now need to be referenced by the module instance.
* You need two sets of `output`s : one in the module definiion and one in the root module.